specification/translation/data-plane/Azure.AI.DocumentTranslation/preview/2024-11-01-preview/openapi.json

```diff
@@ -0,0 +1,1537 @@
+                }
+              ]
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/SupportedFileFormats"
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/Azure.Core.Foundations.ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "String error code indicating what went wrong."
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Returns a list of supported document formats": {
+            "$ref": "./examples/format.json"
+          }
+        }
+      }
+    },
+    "/document:translate": {
+      "post": {
+        "operationId": "DocumentTranslationOperations_DocumentTranslate",
+        "summary": "Submit a single document translation request to the Document Translation service",
+        "description": "Use this API to submit a single translation request to the Document Translation Service.",
+        "produces": [
+          "application/octet-stream",
+          "application/json"
+        ],
+        "consumes": [
+          "multipart/form-data"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/Azure.Core.Foundations.ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/DocumentTranslateBody.sourceLanguage"
+          },
+          {
+            "$ref": "#/parameters/DocumentTranslateBody.targetLanguage"
+          },
+          {
+            "$ref": "#/parameters/DocumentTranslateBody.category"
+          },
+          {
+            "$ref": "#/parameters/DocumentTranslateBody.allowFallback"
+          },
+          {
+            "$ref": "#/parameters/DocumentTranslateBody.translateTextWithinImage"
+          },
+          {
+            "$ref": "#/parameters/Azure.Core.ClientRequestIdHeader"
+          },
+          {
+            "name": "document",
+            "in": "formData",
+            "description": "Document to be translated in the form",
+            "required": true,
+            "type": "file"
+          },
+          {
+            "name": "glossary",
+            "in": "formData",
+            "description": "Glossary-translation memory will be used during translation in the form. ",
+            "required": false,
+            "type": "array",
+            "items": {
+              "type": "string",
+              "format": "binary"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "type": "file"
+            },
+            "headers": {
+              "x-metered-usage": {
```

Comment:
[⛔](https://aka.ms/azapi/guidelines#http-no-x-custom-headers) DO NOT use "x-" prefix for custom headers, unless the header already exists in production [[RFC 6648](https://datatracker.ietf.org/doc/html/rfc6648)].

Reply:
this header already exists in production, hence not updating.